.github/workflows/docs-preview.yml

```diff
@@ -22,16 +22,12 @@ jobs:
         with:
           auto-update-conda: true
           miniforge-variant: Mambaforge
-          channels: conda-forge
-          activate-environment: pyjanitor-dev
```

Comment:
The value of channels and activate-environment use the variable of mkdocs/environment.yml